src/frameworks/nuxt/index.ts

```diff
@@ -30,10 +39,10 @@ export async function discover(dir: string): Promise<{ mayWantBackend: true } |
   const version = nuxtDependency?.version;
   const anyConfigFileExists = await nuxtConfigFilesExist(dir);
 
-  if (!anyConfigFileExists && !nuxtDependency) return;
-  if (version && gte(version, "3.0.0-0")) return { mayWantBackend: true };
+  if (!anyConfigFileExists && !nuxtDependency) return { publicDirectory };
```

Comment:
This should return void no?

Reply:
probably yes, I'll refactor this function as it got messy with my changes